### DISCUSSION TITLE
GrapesJS float plugin with Floating UI

### DISCUSSION THREAD

**Discussion by bgrand-ch** [GrapesJS/grapesjs#5586] [Show and tell]
Hello,

For a personal project and at work, I needed to rethink the UI/UX of GrapesJS to look like [LeadPages](https://www.leadpages.com/platform/landing-page-builder) builder.

Here's one of the plugins I used:
https://github.com/bgrand-ch/grapesjs-float

Full demonstration:
https://github.com/bgrand-ch/grapesjs-float/blob/main/src/example.ts

Playground (Vue 3):
https://github.com/bgrand-ch/grapesjs-float/tree/main/playground/vue

**Comment by bgrand-ch** [comment]
See also https://github.com/GrapesJS/grapesjs/discussions/5587

**Comment by micker** [comment]
very interesting project !!

**Comment by bgrand-ch** [comment]
ℹ️ Some fixes, readme and vue playground updated
